packages/vertexai/src/types/requests.ts

```diff
@@ -77,6 +77,7 @@ export interface GenerationConfig {
   topK?: number;
   presencePenalty?: number;
   frequencyPenalty?: number;
+  responseMimeType?: string;
```

Comment:
Can we add documentation saying what types are allowed? You can use this as the source, you don't have to copy it verbatim: https://source.corp.google.com/piper///depot/google3/google/cloud/aiplatform/master/content.proto;l=233-244;rcl=625322679 You should just be able to put a
/**
* comment
*/
style comment above it, don't need the `@public` tag. We should probably do it for all the properties eventually but that can be done later. This one probably needs a little extra explanation so we should do this one now.

Reply:
Done.